weblab/templates/entities/entity_versions.html

```diff
@@ -21,14 +21,12 @@ <h2>Versions</h2>
 
   <div id="entityversionlist_content">
     <ul>
-    {% for tag, commit in versions %}
+    {% for tags, commit in versions %}
     <li>
       <p>
         <strong>
           <a class="entityversionlink" href="{{ entity|url_version:commit }}">
```

Comment:
This part seems a little broken when I try it... I think the `url_version` filter is having problems. Needs a `str` around the `tags.get` call but also (if the intention is to link to each tag by its name - which seems like a nice idea) the url pattern will need adapting

Reply:
I clearly wasn't thorough enough in my testing of nasty tag names 😞 I've now read `git check-ref-format --help` and tried to make this more robust. Hopefully I've covered every possibility!